### PR TITLE
Cleanup of installation procedure

### PR DESCRIPTION
**Pull request by greatfireball**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/cctb-it-wuerzburg/installation/pull/22" target="_blank">ReviewNinja

**Comment by greatfireball** [comment]
The installation requires the git clone of an internal repository from gitolite server!

Need to provide access to installation process!

File: final_installation/terminal_server/workshop_pollen_metabarcoding/install_software.sh [hunk 1]
```diff
@@ -10,17 +16,11 @@ pip install qiime
...
-#chmod 755 $PWD/usearch8.0.1623_i86linux32
-#ln -fs $PWD/usearch8.0.1623_i86linux32 /usr/local/bin/usearch
+#ea-utils
+apt install --assume-yes --no-install-recommends ea-utils
+
+# install optional software (SeqFilter/usearch) via special package
+cd /tmp
+git clone git@132.187.22.105:alex-summerschool-2016/opt-software
+cd opt-software
+dpkg -i summerschool-opt-software_1.0.0_amd64.deb
```
Comment (iimog) [inline final_installation/terminal_server/workshop_pollen_metabarcoding/install_software.sh:26]: I tested the installation of this deb package in a fresh ubuntu docker container. Both usearch and SeqFilter were properly installed.

Comment (iimog) [inline final_installation/terminal_server/workshop_pollen_metabarcoding/install_software.sh:26]: !completed

**Pull request merged**